src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -52,6 +53,7 @@ case class WorkspaceRequest (
 case class Workspace(
                       namespace: String,
                       name: String,
+                      realm: Option[String] = None,
```

Comment:
Not sure I follow the default value comment. This is an optional field, so we're not giving it a default value.